Sources/Amplitude/Amplitude.m

```diff
@@ -588,16 +601,18 @@ - (void)logEvent:(NSString *)eventType withEventProperties:(NSDictionary *)event
             return;
         }
         if ([eventType isEqualToString:IDENTIFY_EVENT] || [eventType isEqualToString:GROUP_IDENTIFY_EVENT]) {
-            (void) [self.dbHelper addIdentify:jsonString];
+            [self->_identifyBuffer addObject:event];
+            [AMPStorage storeIdentify:jsonString];
         } else {
-            (void) [self.dbHelper addEvent:jsonString];
+            [self->_eventsBuffer addObject:event];
+            [AMPStorage storeEvent:jsonString];
         }
 
         AMPLITUDE_LOG(@"Logged %@ Event", event[@"event_type"]);
 
         [self truncateEventQueues];
```

Comment:
In `truncateEventQueues` we truncate the _eventBuffer when there are too many events, but we didn't do anything to the storage file. Feels like the event will inconsistent. Will this cause some issues?